framework/db/SchemaBuilderTrait.php

```diff
@@ -44,9 +44,9 @@
      * @param integer $length column size or precision definition.
      * This parameter will be ignored if not supported by the DBMS.
      * @return ColumnSchemaBuilder the column instance which can be further customized.
-     * @since 2.0.6
+     * @since 2.0.11
      */
-    public function primaryKey($length = null)
+    public function serial($length = null)
     {
         return $this->getDb()->getSchema()->createColumnSchemaBuilder(Schema::TYPE_PK, $length);
```

Comment:
if you look at mysqls definition for SERIAL (SERIAL is an alias for BIGINT UNSIGNED NOT NULL AUTO_INCREMENT UNIQUE.), this is a bit confusing and perhabs error prone, using the Schema::TYPE_PK.